apps/datahub/src/app/app.component.html

```diff
@@ -1,6 +1,6 @@
 <div
   gnUiSearchRouterContainer="mainSearch"
-  class="selection:bg-primary-lightest selection:text-primary-darker"
+  class="selection:bg-primary-lightest selection:text-primary-darker mb-44 md:mb-96"
```

Comment:
I have looked into it again and it seems that these classes have no effect at all? they could be padding, but then the layout will be strange when looking at a record because the bottom part of the page will be completely white

Reply:
Ok, I remove it.